[2024-08-28T02:18:48.020+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-28T02:18:48.048+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: country_data_pipeline.ingest_data manual__2024-08-28T02:18:43.296410+00:00 [queued]>
[2024-08-28T02:18:48.065+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: country_data_pipeline.ingest_data manual__2024-08-28T02:18:43.296410+00:00 [queued]>
[2024-08-28T02:18:48.066+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-08-28T02:18:48.122+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): ingest_data> on 2024-08-28 02:18:43.296410+00:00
[2024-08-28T02:18:48.131+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=156) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-28T02:18:48.130+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'country_data_pipeline', 'ingest_data', 'manual__2024-08-28T02:18:43.296410+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/welcome_dags.py', '--cfg-path', '/tmp/tmpft5fumw_']
[2024-08-28T02:18:48.133+0000] {standard_task_runner.py:72} INFO - Started process 165 to run task
[2024-08-28T02:18:48.133+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask ingest_data
[2024-08-28T02:18:48.253+0000] {task_command.py:467} INFO - Running <TaskInstance: country_data_pipeline.ingest_data manual__2024-08-28T02:18:43.296410+00:00 [running]> on host 0a02e9b4f651
[2024-08-28T02:18:48.381+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='country_data_pipeline' AIRFLOW_CTX_TASK_ID='ingest_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-28T02:18:43.296410+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-28T02:18:43.296410+00:00'
[2024-08-28T02:18:48.383+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-28T02:18:50.034+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-28T02:18:50.047+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-28T02:18:50.048+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=country_data_pipeline, task_id=ingest_data, run_id=manual__2024-08-28T02:18:43.296410+00:00, execution_date=20240828T021843, start_date=20240828T021848, end_date=20240828T021850
[2024-08-28T02:18:50.119+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-28T02:18:50.152+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-28T02:18:50.165+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
